tests/io/datasets/test_seq.py

```diff
@@ -120,12 +129,217 @@ def test_positive_sync_offset(default_seq, lt_ctx):
 
     result_with_offset = lt_ctx.run_udf(dataset=ds_with_offset, udf=udf)
     result_with_offset = result_with_offset['intensity'].raw_data[
-        :ds_with_offset._meta.image_count - sync_offset
-    ]
+                         :ds_with_offset._meta.image_count - sync_offset
+                         ]
 
     assert np.allclose(result, result_with_offset)
 
 
+def test_array_cropping():
+    start_size = (1024, 1024)
+    crop_to_this = (512, 512)
+    offset = (600, 600)
+    array = np.zeros(start_size)
+    n_array = array_cropping(array, start_size, crop_to_this, offset)
+    assert np.array_equal(n_array, array)
+
+
+def test_xml_excluded_pixels_unbinned():
+    xml_string = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>' \
+                 '<Configuration><PixelSize></PixelSize><DiffPixelSize>' \
+                 '</DiffPixelSize><BadPixels><BadPixelMap Rows="4096" ' \
+                 'Columns="4096"><Defect Rows="2311-2312"/><Defect Rows="3413-3414"/><Defect ' \
+                 'Column="2311"/></BadPixelMap><BadPixelMap Binning="2" ' \
+                 'Rows="2048" Columns="2048"><Defect Rows="1155-1156"/><Defect ' \
+                 'Rows="1706-1707"/></BadPixelMap></BadPixels>' \
+                 '</Configuration>'
```

Comment:
For readability, it would be really nice to have the XML properly formatted. For example like this:

```suggestion
    xml_string = '''
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Configuration>
    <PixelSize></PixelSize><DiffPixelSize></DiffPixelSize>
    <BadPixels>
    <BadPixelMap Rows="4096" Columns="4096">
        <Defect Rows="2311-2312"/>
        <Defect Rows="3413-3414"/>
        <Defect Column="2311"/>
    </BadPixelMap>
    <BadPixelMap Binning="2" Rows="2048" Columns="2048">
        <Defect Rows="1155-1156"/>
        <Defect Rows="1706-1707"/>
    </BadPixelMap>
    </BadPixels>
</Configuration>
    '''
```

This may be a bit more "verbose", but also much better readable than before